packages/terra-brand-footer/examples/index-examples/DefaultBrandFooter.example.jsx

```diff
@@ -0,0 +1,53 @@
+      </svg>
+    }
+    contentBottom={
+      <span>
+        <small>Copyright &copy; 2018 Cerner Corporation.  All Rights Reserved.</small>
```

Comment:
Small nit: We should probably update this phase here and in **_../tests/jest/BrandFooter.test.jsx_** from:
`Copyright &copy; 2018 Cerner Corporation.  All Rights Reserved.`
to:
 `Copyright &copy; 2018 Cerner Innovation, Inc. Licensed under the Apache License, Version 2.0`
\
\
to match the actual License Notice text:
https://github.com/cerner/terra-framework/blob/0e20b84175399d04f257d46d8ae445a980597344/packages/terra-brand-footer/NOTICE#L1-L3